src/main/java/dk/kb/datahandler/kaltura/KalturaUtil.java

```diff
@@ -0,0 +1,110 @@
+package dk.kb.datahandler.kaltura;
+
+import java.io.IOException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.kaltura.client.enums.MediaType;
+
+import dk.kb.datahandler.config.ServiceConfig;
+
+public class KalturaUtil {
+
+
+    private static String DOMS_RADIOTV_PATH;     
+    private static String PRESERVICA_TV_PATH;
+    private static String PRESERVICA_RADIO_PATH;
+    
+  
+    
+   
+    
+    private static final Logger log = LoggerFactory.getLogger(KalturaUtil.class);
+  
+     static {        
+        DOMS_RADIOTV_PATH = ServiceConfig.getConfig().getString("streams.domsRadioTvPath");
+        PRESERVICA_TV_PATH=ServiceConfig.getConfig().getString("streams.preservicaTvPath");
+        PRESERVICA_RADIO_PATH=ServiceConfig.getConfig().getString("streams.preservicaRadioPath");        
+    }
+    
+    
+    /**
+     * Map resourceDescription to Kaltura MediaType
+     * 
+     * @param resourceDescription VideoObject or AudioObject
+     * @return Kaltura MediaType with correct format.
+     */
+    public static MediaType getMediaType(String resourceDescription) {
+        
+        if ("VideoObject".equals(resourceDescription)) {             
+            return MediaType.VIDEO;
+        }
+        else {
+            return MediaType.AUDIO;
+        }                         
+    }
+    
+    /**
+     * Map Kaltura MediaType  to correct flavourParamId
+     * 
+     * @param resourceDescription VideoObject or AudioObject
+     * @return flavourParamId  FlavourId for video or Audio. Depend on the KMC partner ID. Not the same across kaltura environments
+     */
+    public static int getFlavourParamId(MediaType mediaType) {
+        if (MediaType.VIDEO.equals(mediaType)){
+            return ServiceConfig.getConfig().getInteger("kaltura.flavourParamIdVideo");
```

Comment:
The kaltura.flavourParam values does not seem to be present in the behaviour config file (Same in line 59).

Reply:
Added, also used get'er method from ServiceConfig.